.github/workflows/dead_links.workflow.yml

```diff
@@ -14,26 +14,33 @@ on:
         required: true
 
 env:
-  NODE_VERSION: 14
+  NODE_VERSION: 16
 
 jobs:
   dead-links:
     name: Check the dead-links on the repo
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-20.04
```

Comment:
Why 20 instead of 22 ?

Reply:
Because this is the one we used everywhere and we know there are no issue with it.